src/group.rs

```diff
@@ -454,82 +454,82 @@ impl Neg for Element {
     }
 }
 
-impl<'b> MulAssign<&'b Scalar> for Element {
+impl<'b> MulAssign<&'b ark_ed_on_bls12_377::Fr> for Element {
```

Comment:
Since `Fr` is re-exported from the crate root, it's probably cleanest to do `use crate::Fr;` at the top and then use `Fr` directly; then if we were to redefine `Fr`, we only have to change it in one place (in the crate root).